### ISSUE TITLE
Page does not have a descriptive heading - AOS Degree Types

### ISSUE THREAD

**Issue by cory-skeers** [uiowa/uiowa#9541]
Reported by CLAS, but could be an issue for any site with Areas of Study.

> On clas.uiowa.edu, the view pages that show what has been tagged as a certificate or minor are showing up as not having descriptive headings (see attached screenshot).
We are unable to retitle these to make the contents clearer and I don’t think we’re even using links to these anywhere on clas.uiowa.edu. As such, I’m not sure if the issue is that these pages shouldn’t be scanned or if we need to be able to retitle them.
What would you all recommend so we can fix the AA error?

These are views pages generated off of the Areas of Study related taxonomies. Not all of them are flagged (e.g. "Bachelor of Arts" isn't), but others are (e.g. "Certificate").

Examples: https://my2.siteimprove.com/Inspector/998387/A11Y/Page?pageId=68235254452&impmd=73131A19C16C018380A89DB686272039&issue=failed+cantTell&wcag=twopointtwo&decision=none#/sia-r115/failed
https://clas.uiowa.edu/degree-types/certificate

https://my2.siteimprove.com/Inspector/998387/A11Y/Page?pageId=68235305422&impmd=73131A19C16C018380A89DB686272039&issue=failed+cantTell&wcag=twopointtwo&decision=none#/sia-r115/failed
https://clas.uiowa.edu/degree-types/minor

# Proposed solution
Any solution should probably be for the AOS feature and not site-specific. Should also look at all possible instances of views pages with it, and not just degree types.
Further complication as the "degree types" label can be customized by site.
Either disable these pages if they aren't being used, or add a "display" field option to use as the title.
